tools/generate-sources-manifests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Missing Copyright and SPDX-license. See other scripts.

Reply:
Added the copyright and the licence info.